ts/receiver/queuedJob.ts

```diff
@@ -25,6 +29,12 @@ function contentTypeSupported(type: string): boolean {
   return Chrome.isImageTypeSupported(type) || Chrome.isVideoTypeSupported(type);
 }
 
+function isMessageModel(
+  msg: MessageModel | MessageModelPropsWithoutConvoProps
+): msg is MessageModel {
+  return (msg as MessageModel).get !== undefined;
```

Comment:
🧙

Reply:
forgot this was my magic lmfao

Reply:
well I cleaned it up by making it a typeguard, but yeah the `.get !== undefined` was from you :D